src/econ.rs

```diff
@@ -38,7 +39,12 @@ impl Econ {
 
         let raw = self.raw.as_mut().unwrap();
 
-        Ok(raw.auth(password.into().as_str())?)
+        raw.auth(password.into().as_str())
+    }
+
+    /// Change auth message
+    pub fn set_auth_message(&mut self, auth_message: String) {
```

Comment:
Consider using `impl ToString` instead of `String` as argument type

Reply:
Okay

Reply:
> Consider using `impl ToString` instead of `String` as argument type

Okay